ctlearn/data_loading.py

```diff
@@ -375,9 +375,10 @@ def get_metadata(self):
                 'num_selected_telescopes': {tel_type: len(tel_ids) for
                     tel_type, tel_ids in self.selected_telescopes.items()},
                 'num_events_before_cuts_by_class_name': self.num_events_before_cuts_by_class_name,
-                'num_images_before_cuts_by_class_name': self.num_images_before_cuts_by_class_name,
+                'num_images_before_cuts_by_tel_and_class_name': self.num_images_before_cuts_by_tel_and_class_name,
                 'num_events_after_cuts_by_class_name' : self.num_passing_events_by_class_name,
                 'num_images_after_cuts_by_class_name' : self.num_passing_images_by_class_name,
```

Comment:
Shouldn't this be updated to match the new format of the num images before cuts parameter?

Reply:
To be clear, I have not changed the format - I just renamed the metadata field to better reflect its structure.